src/components/inactivePill/InactivePill.jsx

```diff
@@ -0,0 +1,13 @@
+import "./InactivePill.scss"
+import {Badge} from '@patternfly/react-core';
+import React from 'react'
+
+function InactivePill({content}) {
+  return (
+    <Badge key={1} screenReaderText="Unread Messages" className='inactive-pill-container'>
```

Comment:
Instead of creating twi separate components for Active and Inactive pills, you can just add conditional class based on condition of active/inactive.

Reply:
Fixed the issue by conditionally rendering a single pill with different styles according to props passed to the component.